Nodejs/Tests/TestAdapter.Tests/TestProjectFactory.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using static Microsoft.NodejsTools.TestAdapter.TestFrameworks.TestFramework;
+
+namespace TestAdapter.Tests
+{
+    public class TestProjectFactory
```

Comment:
Does this really need to be a stateful class?
In other words, why not just make this class static and have the caller pass in `projectName` where needed?

Reply:
It doesn't need to be stateful. My first thought was to avoid sending multiple parameters but I don't mind making it static. I'll change it as suggested.